### PR TITLE
Create button and input components

### PR DESCRIPTION
**Pull request by rokoc003**
https://tidyapp-pipe-components-2hdapw.herokuapp.com/storybook/?path=/story/button--login-button

### CODE REVIEW

File: client/src/components/Button.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import styled from '@emotion/styled';
+
+const Button = styled.button`
+  background-image: linear-gradient(#fb8777, #ef1846);
+  cursor: pointer;
+  color: #f7efba;
+  border-radius: 10px;
+  box-shadow: 2px 4px 7px #aaaaaa;
+  border: none;
+  font-size: 14px;
```
Comment (JuleFloorson) [inline client/src/components/Button.js:10]: Why don't you use rem here for the font size? You can use this when you defined a font size in  globalstyles :)

File: client/src/components/Input.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import styled from '@emotion/styled';
+
+const Input = styled.input`
+  display: flex;
+  justify-content: center;
+  padding: 2px 8px;
+  color: #bc325b;
+  height: 25px;
```
Comment (JuleFloorson) [inline client/src/components/Input.js:8]: You work here with px for the height. Leon has comment this in a pull req. from Flo. We should work with flex basis, shrink etc. I am Searing for the comment and will send it to you

Comment (JuleFloorson) [reply client/src/components/Input.js:8]: https://github.com/florianGierlichs/mycrypt/pull/24

File: client/src/components/Button.js (hunk 1 above)
Comment (rokoc003) [reply client/src/components/Button.js:10]: Yeees, i forgot that. Thank you 😊

**Pull request merged**